app/src/main/java/org/oneedu/connection/ConnectivityChangeReceiver.java

```diff
@@ -13,20 +11,8 @@
 public class ConnectivityChangeReceiver extends BroadcastReceiver {
     @Override
     public void onReceive(Context context, Intent intent) {
-        ConnectivityManager connectionManager = (ConnectivityManager)context.getSystemService(Context.CONNECTIVITY_SERVICE);
```

Comment:
Make connectivity change receiver just start proxy service and proxy service check network status itself.